local-cli/generator/templates/index.android.js

```diff
@@ -28,9 +27,9 @@ var <%= name %> = React.createClass({
       </View>
     );
   }
-});
+}
 
-var styles = StyleSheet.create({
+const styles = StyleSheet.create({
```

Comment:
Should I still move it to the top?
Is the declaration and usage of `const` and `var` really that different?